extractor.go

```diff
@@ -126,8 +126,20 @@ func (extr *ContentExtractor) GetMetaLanguage(document *goquery.Document) string
 	} else {
 		language = attr[0:idx]
 	}
+	language = strings.ToLower(language)
+
+	// the logic above could pull back garbage string for the language:
+	// as a precaution, check the language against the stopword filter keys to see if we have the corresponding stopword list
+	if _, ok := filters[language]; ok {
```

Comment:
could you make a function like `getBloomFilterForLanguage(lang string) (*bloom.Bloom, bool)` ?

Reply:
ok